data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/IMatrix.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Bundle DataManager API is part of the OrbisGIS platform
+ *
+ * OrbisGIS is a java GIS application dedicated to research in GIScience.
+ * OrbisGIS is developed by the GIS group of the DECIDE team of the
+ * Lab-STICC CNRS laboratory, see <http://www.lab-sticc.fr/>.
+ *
+ * The GIS group of the DECIDE team is located at :
+ *
+ * Laboratoire Lab-STICC – CNRS UMR 6285
+ * Equipe DECIDE
+ * UNIVERSITÉ DE BRETAGNE-SUD
+ * Institut Universitaire de Technologie de Vannes
+ * 8, Rue Montaigne - BP 561 56017 Vannes Cedex
+ *
+ * DataManager API  is distributed under GPL 3 license.
```

Comment:
What about using a LGPL 3 as H2GIS for datamanager ?

Reply:
If we change the license, it will come in an other PR, better submit it as an issue.